### DISCUSSION TITLE
de/guides/ai/

### DISCUSSION THREAD

**Discussion by giscus** [cogneon/lernos#16] [General]
# de/guides/ai/

https://lernos.org/de/guides/ai/

**Comment by rednaxelaesor** [comment]
Der lernOS KI-Leitfaden ist ein beeindruckendes und umfassendes Werk, das sowohl für Anfänger als auch Fortgeschrittene geeignet ist. Er bietet eine klare und strukturierte Einführung in die Welt der Künstlichen Intelligenz mit praxisnahen Beispielen und anschaulichen Erklärungen. Der Leitfaden überzeugt durch hohe Qualität, praxisorientierte Übungen und eine benutzerfreundliche Gestaltung. Er ist sowohl theoretisch fundiert als auch praktisch anwendbar, was ihn zu einem unverzichtbaren Werkzeug für alle macht, die sich mit KI beschäftigen möchten. Ein großes Lob an die Autoren für dieses herausragende Werk!

**Comment by aixotic** [comment]
Als Dozent an einem Weiterbildungsinstitut ist es mir ein Anliegen, mich für Ihre wertvolle Arbeit zu bedanken.
Ich nutze seit drei Jahren KI im Unterricht und sporne meine Schüler dazu an, KI als "Sidekick" zu nutzen.
Ihr Leitfaden hat mir geholfen, das Thema hier im Institut nochmal auf ein anderes Level zu heben.
Mittlerweile nutzen die Teilnehmer die KI in unterschiedlicher Form zum Teil auch ganz gezielt zur Prüfungsvorbereitung auf die Abschlussprüfungen der IHK.
Also auch von mir nochmal ein dickes Lob an die Autoren.

**Reply by simondueckert** [reply]
Hallo aixotic, das freut uns sehr, vielen Dank! Da sind wir direkt auch noch motivierter für die anstehende [Überarbeitung des Leitfadens zu einer Version 1.0](https://community.cogneon.de/t/ueberarbeitung-lernos-ki-leitfaden-zur-version-1-0/4159).
